### DISCUSSION TITLE
"More" checkbox under Image Quality / bitrate settings

### DISCUSSION THREAD

**Discussion by fragtion** [rustdesk/rustdesk#6295] [Q&A]
Hello

There is a "More" checkbox in the settings area for "Default image quality":

![image](https://github.com/rustdesk/rustdesk/assets/2876889/b3e0ef5d-337b-4166-b31f-4187168e8559)

Also when changing the bitrate for an established connection:
![image](https://github.com/rustdesk/rustdesk/assets/2876889/f8245c34-2db3-48ac-bbaa-d77b674ab8df)

The label is very ambiguous. What does this checkbox do exactly?

I assume it allows bitrate to burst higher than what is configured? In that case, shouldn't the checkbox be disabled unless maximum slider value is selected anyway?

Either way I suggest to make the label description more verbose please, as just calling it "More" it's unclear about what it does exactly

**Comment by 21pages** [comment]
Default display options apply to the first connection, otherwise recent configurations will be used, delete the recent session for that peer, and try again.

**Comment by fragtion** [comment]
I understand the difference between using default configuration and recent configuration (I prefer recent configuration for each peer)...

By default, recent connection settings (from the toml) are always used, yeah?

So what I don't understand, is your your explanation (what you mean by "delete the recent session for that peer, and try again."), and when what does the "More" checkbox do exactly when enabled? And what happens when it is disabled?

Thank you

**Reply by 21pages** [reply]
![chrome_4fH2jh6wQB](https://github.com/rustdesk/rustdesk/assets/14891774/8a339065-6e2b-4729-92b0-7a0c032221ae)

![rustdesk_8MCSsbCvZT](https://github.com/rustdesk/rustdesk/assets/14891774/7d9f1f22-e0ce-4ba5-903c-7b5e3b12d246)

**Comment by fragtion** [comment]
Ah OK - so now I think I understand... Checking "More" allows expanding bitrate beyond 100% (to 2000% max). I think we were misunderstanding each other because I already understood recent vs default connection and how it works, I just wanted to understand the "more" checkbox.

Thank you.

So now I have another question:

~1) What is the kbps/Mbps correlation or ratio with the Bitrate percent ("%") ?   - Is this documented anywhere?~
_Edit: Ok I see if I enable "Show quality monitor", it reveals the "Target Bitrate" in the quality monitor overlay._
**Suggestion:** It would be nice to show this target bitrate in the same dialog as the Bitrate slider, so the user can see not only a percent, but also the target kbps before establishing any connections

**Reply by southerndoc** [reply]
> Ah OK - so now I think I understand... Checking "More" allows expanding bitrate beyond 100% (to 2000% max). I think we were misunderstanding each other because I already understood recent vs default connection and how it works, I just wanted to understand the "more" checkbox.
> 
> Thank you.
> 
> So now I have another question:
> 
> ~1) What is the kbps/Mbps correlation or ratio with the Bitrate percent ("%") ? - Is this documented anywhere?~ _Edit: Ok I see if I enable "Show quality monitor", it reveals the "Target Bitrate" in the quality monitor overlay._ **Suggestion:** It would be nice to show this target bitrate in the same dialog as the Bitrate slider, so the user can see not only a percent, but also the target kbps before establishing any connections

Have you noticed quality improvement between 1000% and 2000%?
